dev-python/mkdocs-bootstrap/mkdocs-bootstrap-1.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Bootstrap theme for MkDocs"
+HOMEPAGE="https://www.mkdocs.org https://github.com/mkdocs/mkdocs-bootstrap"
+SRC_URI="https://github.com/mkdocs/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="doc"
+
+DEPEND="dev-python/mkdocs"
```

Comment:
Is this a BDEPEND?

Reply:
I don't think so, as I understand it, it is a theme for mkdocs. mkdocs is required as BDEPEND to build mkdocs-bootstrap[doc] but also as RDEPEND to actually use the theme (a theme for the docs compiler without the actual compiler is a bit useless)